doc/releases/release-notes-4.0.rst

```diff
@@ -266,6 +319,18 @@ Boards & SoC Support
 
   * Added Thingy53 forwarding of network core pins to network core for SPI peripheral (disabled
     by default) including pin mappings.
+  * Added uart, flexio pwm, flexio spi, watchdog, flash, rtc, i2c, lpspi, edma, gpio, acmp, adc and lptmr support
+    to NXP ``frdm_ke17z`` and ``frdm_ke17z512``
+  * Enabled support for MCUmgr on NXP boards
+  * Added documentation for SEGGER SystemView support  for all NXP boards with an SoC supporting ECC
+  * Enabled MCUboot, FlexCAN, LPI2C, VREF, LPADC and timers (TPM, LPTMR, counter, watchdog) on NXP ``frdm_mcxw71``
+  * Enabled MRT, lvgl, I2C and XIP (FlexSPI) for NXP ``frdm_mcxn947``.
+  * Removed CONFIG_PINCTRL from the defconfig of many NXP boards
```

Comment:
wasn't this also another effort done for all boards in tree, not just nxp

Reply:
it looks like in the pinctrl driver section of these release notes, its mentioned but with only STM32: in front of the bullet, maybe just delete that STM32: before the line instead of making this new bullet under nxp platform